torch/nn/functional.py

```diff
@@ -3912,7 +3912,11 @@ def interpolate(input: Tensor, size: Optional[int] = None, scale_factor: Optiona
             ]
         else:
             assert scale_factors is not None
-            output_size = [int(math.floor(float(input.size(i + 2)) * scale_factors[i])) for i in range(dim)]
+            from ..fx.experimental.symbolic_shapes import sym_int, sym_float
+            output_size = [
+                sym_int(math.floor(sym_float(input.size(i + 2)) * scale_factors[i]))
```

Comment:
This is failing a bunch of TS unittests...  


```
Traceback (most recent call last):
  File "/scratch/bahuang/work/repos/pytorch/test/quantization/jit/test_quantize_jit.py", line 2994, in test_general_value_ops
    m = torch.jit.script(M())
  File "/scratch/bahuang/work/repos/pytorch/torch/jit/_script.py", line 1286, in script
    return torch.jit._recursive.create_script_module(
  File "/scratch/bahuang/work/repos/pytorch/torch/jit/_recursive.py", line 477, in create_script_module
    return create_script_module_impl(nn_module, concrete_type, stubs_fn)
  File "/scratch/bahuang/work/repos/pytorch/torch/jit/_recursive.py", line 543, in create_script_module_impl
    create_methods_and_properties_from_stubs(concrete_type, method_stubs, property_stubs)
  File "/scratch/bahuang/work/repos/pytorch/torch/jit/_recursive.py", line 394, in create_methods_and_properties_from_stubs
    concrete_type._create_methods_and_properties(property_defs, property_rcbs, method_defs, method_rcbs, method_defaults)
  File "/scratch/bahuang/work/repos/pytorch/torch/jit/_script.py", line 1407, in _get_overloads
    _compile_function_with_overload(overload_fn, qual_name, obj)
  File "/scratch/bahuang/work/repos/pytorch/torch/jit/_script.py", line 1381, in _compile_function_with_overload
    fn = torch._C._jit_script_compile_overload(
RuntimeError: 
object has no attribute symbolic_shape:
  File "/scratch/bahuang/work/repos/pytorch/torch/nn/functional.py", line 3916
            assert scale_factors is not None
            output_size = [
                torch.fx.experimental.symbolic_shape.sym_int(
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
                    math.floor(torch.fx.experimental.symbolic_shape.sym_float(input.size(i + 2)) * scale_factors[i]))
                for i in range(dim)
'interpolate' is being compiled since it was called from 'M.forward'
  File "/scratch/bahuang/work/repos/pytorch/test/quantization/jit/test_quantize_jit.py", line 2968
                x = x.mean([2, 3], True)
                # interpolate node will introduce 3 quantize_per_tensor ops
                x = F.interpolate(x, 4, mode="nearest")  # interpolate node
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
                x = F.upsample(x, (32, 32))  # interpolate node
                x = F.upsample_nearest(x, (32, 32))  # interpolate node

============================================================================================================================================================================================== short test summary info ==============================================================================================================================================================================================
FAILED test/quantization/jit/test_quantize_jit.py::TestQuantizeJitOps::test_general_value_ops - RuntimeError: 
```